README.md

```diff
@@ -654,6 +659,15 @@ Switched to a new branch 'daves'
 
 This will give you a local copy of the branch `daves`, and any update that has been pushed will also show up remotely.
 
+<a name="i-want-to-create-a-new-remote-branch-from-current-local-one"></a>
+### I want to create a new remote branch from current local one
+
+```sh
+$ git config --global push.default current
```

Comment:
I'm not sure this needs to be a global config setting - it could be a local one. That seems better practice, to me.